packages/driver/src/dom/elements/complexElements.ts

```diff
@@ -290,6 +290,12 @@ export const isScrollable = ($el) => {
     return false
   }
 
+  const documentElement = $document.getDocumentFromElement(el).documentElement
+
+  if (el === documentElement) {
```

Comment:
All the other scenarios are documented in this function, suggest documenting this one as well.